deepmd/fit/polar.py

```diff
@@ -511,17 +517,23 @@ def build (self,
         return tf.reshape(outs, [-1])
     
     def init_variables(self,
-                       model_file: str
+                       graph: tf.Graph,
+                       graph_def: tf.GraphDef,
+                       suffix : str = "",
     ) -> None:
         """
-        Init the fitting net variables with the given frozen model
+        Init the fitting net variables with the given dict
 
         Parameters
         ----------
-        model_file : str
-            The input frozen model file
+        graph : tf.Graph
+            The input frozen model graph
+        graph_def : tf.GraphDef
+            The input frozen model graph_def
+        suffix : str
+            suffix to name scope
         """
-        self.polar_fitting.init_variables(model_file)
+        self.polar_fitting.init_variables(graph=graph, graph_def=graph_def, suffix=suffix)
```

Comment:
It's unclear why this class does not inherit from `self.polar_fitting` directly...

Reply:
The class `GlobalPolarFitting` is deprecated. `PolarFitting` can to all the things that the `GlobalPolarFitting` is designed to do.